components/bill/invoice/helpers.go

```diff
@@ -0,0 +1,13 @@
+package invoice
+
+import (
+	"github.com/invopop/gobl"
+	"github.com/invopop/gobl.html/components/regimes/es"
+)
+
+// HasHeaderQR returns a boolean indicating whether the envelope has some QR codes to be displayed in the header or not.
+func HasHeaderQR(env *gobl.Envelope) bool {
```

Comment:
Does this need to be a public method?

Reply:
It is a helper used on invoice.templ. Should I just move this method to invoice?